test/scripts/jenkins_xpack.sh

```diff
@@ -23,11 +23,11 @@ echo ""
 echo ""
 
 # FAILING: https://github.com/elastic/kibana/issues/44250
-# echo " -> Running jest contracts tests"
-# cd "$XPACK_DIR"
-# SLAPSHOT_ONLINE=true CONTRACT_ONLINE=true node scripts/jest_contract.js --ci --verbose
-# echo ""
-# echo ""
+echo " -> Running jest contracts tests"
```

Comment:
are the tests stable enough to re-enable them in CI for our feature branch?

Reply:
The tests themselves are and have been stable. The issue was a timeout on CI waiting for Kibana to start when CI was under load. Only time will tell but I ran the tests on a server I put under reasonable load then ran the tests and it seems to be enough.